lib/plaid/products/item.rb

```diff
@@ -226,5 +226,17 @@ def delete(access_token)
       payload = { access_token: access_token }
       @client.post_with_auth('item/delete', payload)
```

Comment:
I have just a bit of background on the justification for this change aside from wanted to deprecate `delete`, but is there any chance we'd want to either get rid of the `delete` function and bump the version or change the `delete` function to call `remove` instead? We could write a small shim that converts a response from `remove` to look like the response from `delete` to maintain backward compatibility.

Reply:
Great idea, @edmundloo -- planning to do this in a follow up pull request alongside all of the Plaid client libraries!

Reply:
Sounds good! :)